### ISSUE TITLE
Feature Request: Blacklist "Bad" weathers during cutscenes/zone intros unless it's part of the cutscene.

### ISSUE THREAD

**Issue by KageCM** [NightmareXIV/Weatherman#37]
I think we've all experienced at some point, a cutscene or zone into ruined because of fog, heavy rain, or it's too dark at night. Would it be possible to make it so disruptive or obstructive weather wasn't present during cutscenes or zone intros? Possibly change the time of day too? Unless the cutscene itself sets specific ones itself. If that isn't possible maybe make a button to toggle Weatherman on/off? 

Either way thanks for the great mod.

**Comment by Limiana** [comment]
Hey, there's a setting in Weatherman: 
![image](https://github.com/user-attachments/assets/7f8a27f8-266e-48fb-9d07-4681446df355)
seems like you want to disable it

**Comment by KageCM** [comment]
> Hey, there's a setting in Weatherman: ![image](https://private-user-images.githubusercontent.com/5073202/371688422-7f8a27f8-266e-48fb-9d07-4681446df355.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjc0Nzg0MTEsIm5iZiI6MTcyNzQ3ODExMSwicGF0aCI6Ii81MDczMjAyLzM3MTY4ODQyMi03ZjhhMjdmOC0yNjZlLTQ4ZmItOWQwNy00NjgxNDQ2ZGYzNTUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDkyNyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA5MjdUMjMwMTUxWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9OWM0NGI2N2FhNTQ1YTNhNjMzZTA5OTBjZjdiNGYyYWQ4MDU2MDM3NGM3YTNmNjNiODJmNTcwZmQ1ZTFiMTUyYSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.CKw62tZUzyMvsHIekSQmoaDV1LuJKduG-UG_z83R9YI) seems like you want to disable it

Hi. I saw that option but I was kind of looking more for something to automatically stop or blacklist only certain weathers during cutscenes unless the cutscene itself enforces specific conditions. SE does this for some cutscenes but most you're stuck with whatever the current is which I'm usually fine with. And I know that this may be difficult so another option I was thinking about was a button or keybind available during cutscenes to toggle it on or off.

**Comment by Limiana** [comment]
Do you mean you want to blacklist certain weathers only in cutscenes and not anywhere else?

**Comment by KageCM** [comment]
> Do you mean you want to blacklist certain weathers only in cutscenes and not anywhere else?

Yes. The weather doesn't bother me for the most part. It's only during cutscenes and the link where I think it can ruin the experience. I've also seen many posts on Reddit poking fun at particular scenes in Shadowbringers where you're supposed to either look at the sky or the view across the lake and instead there's a wall of fog. Quite a few streams have also instances as such to the point of referring to it as "the curse of the fog" or something like that.

**Comment by Limiana** [comment]
I see, well it's not possible for now but I will make it possible in future

**Comment by KageCM** [comment]
> I see, well it's not possible for now but I will make it possible in future

Awesome. Great to hear. From the very minor coding experience from way back I understand that these things can seem simple but end up being very difficult. So I will look forward to it if/when it's possible. Thank you for your time.
